components/org.wso2.carbon.identity.captcha/src/main/java/org/wso2/carbon/identity/captcha/util/CaptchaUtil.java

```diff
@@ -258,8 +260,19 @@ public static boolean isValidCaptcha(String reCaptchaResponse) throws CaptchaExc
         try {
             try (InputStream in = entity.getContent()) {
                 JsonObject verificationResponse = new JsonParser().parse(IOUtils.toString(in)).getAsJsonObject();
+                if (log.isDebugEnabled()) {
+                    if (verificationResponse.get("score") != null) {
```

Comment:
Assign verificationResponse.get("score") to a variable in all other places

Reply:
Is this ok?